scipy/stats/_qmc.py

```diff
@@ -1284,11 +1331,21 @@ class LatinHypercube(QMCEngine):
 
             .. versionadded:: 1.8.0
 
-        seed : {None, int, `numpy.random.Generator`}, optional
-            If `seed` is an int or None, a new `numpy.random.Generator` is
-            created using ``np.random.default_rng(seed)``.
-            If `seed` is already a ``Generator`` instance, then the provided
-            instance is used.
+        rng : `numpy.random.Generator`, optional
+            Pseudorandom number generator state. When `rng` is None, a new
+            `numpy.random.Generator` is created using entropy from the
+            operating system. Types other than `numpy.random.Generator` are
+            passed to `numpy.random.default_rng` to instantiate a ``Generator``.
+
+            .. versionchanged:: 1.15.0
+
+                As part of the `SPEC-007 <https://scientific-python.org/specs/spec-0007/>`_
+                transition from use of `numpy.random.RandomState` to
+                `numpy.random.Generator`, this keyword was changed from `seed` to
+                `rng`. For an interim period, both keywords will continue to work, although
```

Comment:
This is the source of the lint failure. I'll fix it when we have a reviewer for this. Reviewer: mind if I dedent this whole docstring? The whole thing one level higher than necessary.